pallets/subtensor/src/lib.rs

```diff
@@ -1527,12 +1534,20 @@ pub mod pallet {
 		}
 
 		#[pallet::call_index(46)]
+		#[pallet::weight((Weight::from_ref_time(13_000_000)
+		.saturating_add(T::DbWeight::get().reads(1))
+		.saturating_add(T::DbWeight::get().writes(1)), DispatchClass::Operational, Pays::No))]
+		pub fn sudo_set_quadratic_voting_power( origin:OriginFor<T>, netuid: u16, quadratic_voting_power: u16 ) -> DispatchResult {
+			Self::do_sudo_set_quadratic_voting_power( origin, netuid, quadratic_voting_power )
+		}
+
+		#[pallet::call_index(47)]
 		#[pallet::weight((0, DispatchClass::Operational, Pays::No))]
 		pub fn sudo_set_total_issuance(origin: OriginFor<T>, total_issuance: u64 ) -> DispatchResult {
 			Self::do_set_total_issuance(origin, total_issuance)
 		}
 		
-		#[pallet::call_index(47)]
```

Comment:
@camfairchild may these call_index values be modified like this?

Reply:
You should assign the next call index to your new sudo call. We should avoid reassigning call indices, as this is effectively the API call name

Reply:
Noted, fixed, thx.